pysoa/common/transport/asgi/core.py

```diff
@@ -96,6 +95,10 @@ def __len__(self):
 
 class SentinelRedisChannelLayer(RedisChannelLayer):
 
+    def __init__(self, *args, **kwargs):
+        super(SentinelRedisChannelLayer, self).__init__(*args, **kwargs)
+        self.ring_size = len(self._connection_list)
+
     def _generate_connections(self, redis_kwargs):
         return SentinelMasterConnectionList(self.hosts, redis_kwargs)
```

Comment:
Should this method update the ring size as well (i.e. not only on initialization)?